Source/storm/storm.cpp

```diff
@@ -215,6 +215,41 @@ void setIniFloat(const char *keyname, const char *valuename, float value)
 	getIni().SetDoubleValue(keyname, valuename, value);
 }
 
+/**
+ * @brief Check item exists in comma-sparated list
+ */
+bool IniFound(const char *list, const char *item)

```

Comment:
1. There is no need for this to be case-insensitive.
2. Instead of calling `IniFound` for every item, it'd be better to loop over the comma-separated items once.

Reply:
Case-intensively is not must but it helps. 
For long list like spell names, looping is must. But five items are not worth it.